Makefile

```diff
@@ -47,10 +50,10 @@ ifndef IN_NIX_SHELL
 		echo "Configuring Nix shell..."; \
 		if ! command -v "nix" >/dev/null 2>&1; then \
 			. ~/.nix-profile/etc/profile.d/nix.sh; \
-			NIX_CONF_DIR=$(PWD)/scripts/lib/setup/nix \
+			NIX_CONF_DIR=$(PWD)/nix \
 				nix-shell; \
 		else \
-			NIX_CONF_DIR=$(PWD)/scripts/lib/setup/nix \
+			NIX_CONF_DIR=$(PWD)/nix \
```

Comment:
Why don't we set this somewhere at the top of the `Makefile`? I don't see any reason for it not to be set.

Reply:
Will do that. By the way will remove __toolversion, since that's not used anymore in the Makefile.